app/Repositories/Eloquents/AbstractEloquentRepository.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace App\Repositories\Eloquents;
+
+abstract class AbstractEloquentRepository
+{
+	abstract public function model();
+}
\ No newline at end of file
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Files.EndFileNewline.NoneFound</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/AbstractEloquentRepository.php</br>Expected 1 newline at end of file; 0 found